### PR TITLE
citations: Add new coffee paper

### PR DESCRIPTION
**Pull request by FabioLuporini**

### CODE REVIEW

**Comment by wence-** [comment]
OK.

**Comment by wence-** [comment]
Given the citations changes in coneoproject/COFFEE#77, you should remove the automated registration of the coffee papers that happen in firedrake.  That requires removing the lines around L66 in parameters.py.

**Comment by wence-** [comment]
You should also add the bibtex entry to `docs/source/_static/bibliography.bib` and modify the text in `docs/source/publications.rst` where we talk about how to cite firedrake "manually".

**Comment by FabioLuporini** [comment]
Done what you say. Also rebased on master

File: docs/source/publications.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ when using quadrilateral meshes, please also cite :cite:`Homolya2016`.
 If your work relies on the kernel-level performance optimisations that
 Firedrake performs using `COFFEE
 <http://github.com/coneoproject/COFFEE>`_, please cite the
-COFFEE paper :cite:`Luporini2015`.
+COFFEE paper :cite:`Luporini2015` and :cite:`Luporini2016`.
```
Comment (wence-) [inline docs/source/publications.rst]: s/paper/papers/

**Comment by FabioLuporini** [comment]
fixed di 's'

**Pull request merged**